tests/test_schunk_proxy.py

```diff
@@ -0,0 +1,80 @@
+#######################################################################
```

Comment:
I think a more consistent name for this module would be `test_proxy_schunk.py`

Reply:
Fixed in 42c27b8bc76bc1c4ed98a708df63fec788dd1b0a.